fapi-client/src/test/scala/com/gu/facia/api/integration/IntegrationTest.scala

```diff
@@ -73,12 +74,13 @@ class IntegrationTest extends FreeSpec with ShouldMatchers with ScalaFutures wit
 
   "backfill" - {
     val collection = Collection(
-      CollectionId("uk/business/regular-stories"),
+      "uk/business/regular-stories",
       "economy",
       Nil,
       None,
-      Some("updatedBy"),
-      Some("updatedBy@example.com"),
```

Comment:
Talk to @robertberry about this. He literally only just switched them over to Options and now you're switching them back.